themes/_carfactory.scss

```diff
@@ -2,6 +2,7 @@
 //
 
 // Inherit from original theme
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
```

Comment:
I did this in order to make sui-components work, maybe carfactory theme should load the compat vars its uses

@davidmartin84

Reply:
This should be done in the cf-theme, not here